src/utils/api.js

```diff
@@ -26,4 +26,17 @@ export const patchArticle = (article_id, vote) => {
     return newsAPI.patch(`/articles/${article_id}`, vote).then(( { data } ) => {
         return [data.updatedArticle]
     })
-}
\ No newline at end of file
+}
+
+export const getTopics = () => {
+    return newsAPI.get(`/topics`).then(( {data} ) => {
+        return data.topics
+    })
+  }
+
+  export const getQueriedArticles = (sortByQuery) => {
+    return newsAPI.get(`/articles?topic=${sortByQuery}`).then(( { data } ) => {
+        return data.articles
+      });
+  };
```

Comment:
This does not need to be a seperate function.

I would recommend looking into axios params, and refactoring your original getArticles so it can list all articles by default as well as filtering articles by topic.